test/replication/gh-3055-promote-wakeup-crash.result

```diff
@@ -38,7 +41,7 @@ f:cancel()
 f:join()
  | ---
  | - false
- | - fiber is cancelled
+ | - 'Not enough peers connected to start elections: 1 out of minimal required 2'
```

Comment:
It seems the gh-3055 test doesn't make any sense anymore. Let's simply drop it (and mention in the commit message why)